beta-src/src/components/ui/WDPillScroller.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { Box, ButtonGroup } from "@mui/material";
+import WDScrollButton from "./WDScrollButton";
+import { gameStateProps } from "../../interfaces/PhaseScroll";
```

Comment:
let's have the props in the component i think it's better for people looking at the component to see the props

Reply:
@jchoi166 as Steve mentioned when it comes to component props we don't need those in a separate file unless you think they'll need to be re-used elsewhere - which in the case of component specific props is highly unlikely.

Reply:
@jchoi166 as Steve mentioned when it comes to component props we don't need those in a separate file unless you think they'll need to be re-used elsewhere - which in the case of component specific props is highly unlikely.